internal/contracts/manager.go

```diff
@@ -77,7 +77,7 @@ func NewContractManager(ctx context.Context, database database.Plugin, publicSto
 	return &contractManager{
 		database:          database,
 		txHelper:          txcommon.NewTransactionHelper(database),
-		publicStorage:     publicStorage,
+		sharedStorage:     sharedStorage,
```

Comment:
I'm surprised there are no renames outside of the constructor. Is this even used?